frontend/src/metabase/query_builder/components/notebook/NotebookDataPicker/NotebookDataPicker.tsx

```diff
@@ -57,6 +58,30 @@ export function NotebookDataPicker({
     [query, stageIndex, table],
   );
 
+  const pickerInfo = useMemo(
+    () => table && Lib.pickerInfo(query, table),
+    [query, table],
+  );
+
+  const getUrl = () => {
```

Comment:
nit: it might be just a bit better to move `getUrl` outside of the component body to a separate function in the same file and pass `query`, `table` directly.

Reply:
https://github.com/metabase/metabase/pull/46228/files#diff-bf7d479453077aae4f02f7317c728994a28f5ee338918c18be1bb7efb8d2c478R76